maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -198,6 +202,18 @@ public void executeTask() throws MojoExecutionException, MojoFailureException {
 						+ buildDirectory.getAbsolutePath() + "\n");
 	}
 
+	private DistroProperties handleParentDistroProperties(Artifact distroArtifact, File buildDirectory, DistroProperties distroProperties) throws MojoExecutionException {
+		if (StringUtils.isNotBlank(distroArtifact.getArtifactId()) && StringUtils.isNotBlank(distroArtifact.getGroupId()) && StringUtils.isNotBlank(distroArtifact.getVersion())) {
+			Server tempServer = new Server.ServerBuilder().build();
```

Comment:
Is there a way we could de-couple this from the server setup? It seems like the only real reason we pass this in is for the server directory so maybe `resolveParentArtifact()` and `getArtifactProperties()` could be updated to just take a directory as an argument?

Reply:
I updated the PR with these changes.